src/test/java/io/strimzi/kafka/metrics/DataPointSnapshotBuilderTest.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.metrics;
+
+import io.prometheus.metrics.model.snapshots.InfoSnapshot;
+import io.prometheus.metrics.model.snapshots.Labels;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DataPointSnapshotBuilderTest {
+
+    @Test
+    public void testNewLabelDuplicateWarning() {
```

Comment:
I think a better name would be something like `testCollidingNewLabelIsIgnored`

Reply:
Changed to `testCollidingNewLabelIsIgnored`